backend/mock/service/chaos/experimentation/experimentstoremock/simplestorer.go

```diff
@@ -97,6 +98,14 @@ func (s *SimpleStorer) GetListView(ctx context.Context) ([]*experimentation.List
 func (s *SimpleStorer) RegisterTransformation(transformation experimentstore.Transformation) error {
 	return nil
 }
+func (s *SimpleStorer) GetRuntimeGenerator(configTypeUrl string) (*experimentstore.RuntimeGeneration, error) {
```

Comment:
this objects starts to have a ton of respnsibilities - view related fetches, crud requests and now runtime generators. We may need to think about how to break it up - just a general comment- not saying that this has to be done in this PR.